release/src/version-helpers.unit.spec.ts

```diff
@@ -0,0 +1,277 @@
+import {
+  isValidVersionString,
+  getOSSVersion,
+  getEnterpriseVersion,
+  isEnterpriseVersion,
+  isRCVersion,
+  getVersionType,
+  getReleaseBranch,
+  isLatestVersion,
+} from "./version-helpers";
+
+describe("version-helpers", () => {
+  describe("isValidVersionString", () => {
+    const validCases = [
+      "v0.75.2.3",
+      "v1.75.2.3",
+      "v0.75-RC1",
+      "v1.75-RC2",
+      "v0.45-rc99",
+      "v1.45-RC99",
+      "v0.75-rc0",
+      "v0.75.2",
+      "v1.11",
+      "v1.75",
+      "v1.75.2",
+      "v1.9",
```

Comment:
Not sure we should accept releases without the explicit `.0` minor version.

```diff
- v1.9
+ v1.9.0
- v1.11
+ v1.11.0
```

Reply:
Good call 👍  I updated the regex and tests to enforce the .0 for major releases